base/rational.jl

```diff
@@ -272,7 +272,7 @@ signbit(x::Rational) = signbit(x.num)
 copysign(x::Rational, y::Real) = unsafe_rational(copysign(x.num, y), x.den)
 copysign(x::Rational, y::Rational) = unsafe_rational(copysign(x.num, y.num), x.den)
 
-abs(x::Rational) = Rational(abs(x.num), x.den)
+abs(x::Rational) = Rational(checked_abs(x.num), x.den)
```

Comment:
```suggestion
abs(x::Rational) = unsafe_rational(checked_abs(x.num), x.den)
```
I’m surprised `unsafe_rational` wasn’t used here before, it should be safe and improves performance significantly.

Reply:
I understand that it would safe nevertheless, but can you please explain why it improves performance? I couldn't find the reason.